src/llm_vm/onsite_llm.py

```diff
@@ -3,6 +3,7 @@
 
 import abc
 from abc import ABC,abstractmethod
+import openai
 
```

Comment:
philosophically, theres no reason we need put the openai wrapper here, but its fine for now